collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 25
+update_every = 1
+
+ORDER = [
+    'alarms',
+]
+
+CHARTS = {
+    'alarms': {
+        'options': ['alarms', 'Alarms', 'status', 'alarms', 'alarms.status', 'line'],
```

Comment:
Could add alarm statuses mapping to the title, for instance

`Alarms statuses (0: CLEAR, ...`

Reply:
added a little bit of text to the overview/description part in dashboard_info.js